tools/xtask/src/build.rs

```diff
@@ -184,9 +184,13 @@ fn build_twizzler<'a>(
         return Ok(None);
     }
     crate::print_status_line("collection: userspace", Some(build_config));
+    // the currently supported build target triples
+    // have a value of "unknown" for the machine, but
+    // we might specify a different value for machine
+    // on the cli for conditional compilation
     let triple = Triple::new(
         build_config.arch,
-        build_config.machine,
+        crate::triple::Machine::Unknown,
```

Comment:
Why the change to the hardcoded Unknown?

Reply:
In the case of ARM, when compiling for a hardware target, the machine value is specified. So for QEMU we use `virt` and for something like a raspberry pi, we might use something like `rpi4`. When it comes down to compiling the user space part, we need to tell the compiler to use `aarch64-unknown-twizzler`. Everything else matches up except that part.